### PR TITLE
Clean up some HTML (or JSX) in the Armory Week page

### PR DESCRIPTION
**Pull request by yuki24**
This PR includes the cleanup work below:

 * Clean up HTML (or JSX) by throwing reaction components in
 * Rename the `<Cta>` component to `<BannerPopUp>`
 * Suppress a warning `warning.js:33 Warning: Expected server HTML to contain a matching <div> in <div>` coming from `ReactDOM.hydrate`

Once this is merged I'll send another PR with cfb8b8671 that addresses [AR-116](https://artsyproduct.atlassian.net/browse/AR-116).

### CODE REVIEW

File: desktop/apps/armory_week/client/initializer.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+import { BannerPopUp } from 'desktop/apps/armory_week/components/BannerPopUp'
 
 export const init = () => {
   const bootstrapData = window.__BOOTSTRAP__
   const { ctaTitle, ctaImageUrl, overlayModalTitle, overlayModalImageUrl } = bootstrapData.bannerPopUp
 
-  ReactDOM.hydrate(
-    <Cta {...{ ctaTitle, ctaImageUrl, overlayModalTitle, overlayModalImageUrl }} />,
+  ReactDOM.render(
+    <BannerPopUp {...{ ctaTitle, ctaImageUrl, overlayModalTitle, overlayModalImageUrl }} />,
```
Comment (yuki24) [inline desktop/apps/armory_week/client/initializer.js]: @kanaabe I wasn't sure what a better name could be... open to suggestions.

**Comment by damassi** [comment]
@yuki24 

> Suppress a warning warning.js:33 Warning: Expected server HTML to contain a matching <div> in <div> coming from ReactDOM.hydrate

What did you do to suppress?

**Comment by yuki24** [comment]
I just replaced `ReactDOM.hydrate` with `ReactDOM.render`: 8d40e46

The `<ArmoryWeekPage>` component gets rendered on the server-side while the `<BannerPopUp>` (that has some `onClick` events) gets rendered on the client-side. I simply copied the server-side rendering code to the client-side (which I think was not right) so I assumed `ReactDOM.render` is good, but am I missing something?

**Comment by damassi** [comment]
Ah yeah, if rendering the same component on both the client and the server then `rehydrate` should be used but if you're just bootstrapping data and rendering a different component `render` is fine.

**Pull request merged**